deploy/docs/Performance.md

```diff
@@ -0,0 +1,57 @@
+# Performance
+
+For larger or more volatile loads, we recommend [enabling Fluentd autoscaling](./Best_Practices.md#Fluentd-Autoscaling), as this will allow Fluentd to automatically scale to support your data volume. However, the following recommendations and corresponding examples will help you get an idea of the resources required to run collection on your cluster.
+
+## Recommendations
+
+1. At least **8 Fluentd-logs** pods per **1 TB/day** of logs.
+2. At least **4 Fluentd-metrics** pods per **120k DPM** of metrics.
```

Comment:
Hey @samjsong , have we written 4 Fluentd-metrics by considering the scrape interval as well? Because I want to understand the numbers 120k and 4 fluentd metrics.
AFAIR, one fluentd-metrics is able to support around 60K DPM, so if we are considering the scrape interval of 30sec, then this 4 fluentd is fine but if now, can you tell me the number calculation of 4 fluentd metrics and 120K DPM

Reply:
Right, since from Sumo backend perspective the customer is billed for 120k DPM rather than for 60k DPM at 30s scrape interval, I thought it would make it easier to understand if we kept just one number for the DPM